.github/workflows/k8s-autodiscovery-test.yml

```
@@ -58,4 +58,4 @@ jobs:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
-          slackChannel: "#observablt-bots"
+          slackChannel: "#obs-cloud-monitoring"
```

```suggestion
          slackChannel: "#cloudnative-sdh-alarm"
```

The #obs-cloud-monitor channel is a wider channel for cloud issues. Maybe this alarms can add too much noise

I would say to use #cloudnative-sdh-alarm, an "internal channel" of former cloudnative team that was used to track sdhs and various automated notifications.

@tommyers-elastic , @ChrsMark any ideas?